addons/polardbx/values.yaml

```diff
@@ -21,7 +21,7 @@ images:
   polardbx:
     pullPolicy: IfNotPresent
     # Repo of polardbx default images. Default is polardbx.
-    repository: polardbx
+    repository:
```

Comment:
refine the annotation

Reply:
it's correct .default value is still polardbx
![image](https://github.com/user-attachments/assets/b136aac9-b4f9-4acc-878f-fb6953b78b14)